### DISCUSSION TITLE
Sharing expenses between flatmates based on a percentage

### DISCUSSION THREAD

**Discussion by DesertCookie** [firefly-iii/firefly-iii#8250] [Firefly III]
Does such a thing exist in Firefly III?

About my situation:

Living in a flat of three, we decided to share food expenses. However, we eat quite differently and thus have decided on an uneven split of 40- 35-25%. So far, it's been a hassle for me to input all of our expenditures into a spreadsheet, then calculate who has payed more than their share and thus has a credit, then doing the same for those who contributed too little and how much they own whom.

I have yet to find a free app that supports uneven splits and calculations such as these. _Honestly, I forget how to operate my spreadsheet every six months once I come around to go through our receipts of the last half year or so and have to think twice about whether my calculations are correct or not._

**Comment by JC5** [comment]
I'm afraid not, but I remember apps like these existing. Something like Splitwise perhaps?

**Reply by DesertCookie** [reply]
I think I made a feature request about this to Splitwise some while back. Perhaps they've been quick and implemented it already (though I doubt it, as it clearly doesn't seem to be something many people use).

**Reply by jordantoth** [reply]
Splitwise can do this, though you have to set it on each transaction. When you enter an expense, by default it is labelled as "Paid by [name] and split Equally". If you click the word "Equally" you can view several different ways to split the cost. It sounds like your use case would be "Split by shares", where one person is paying 40 shares, one is paying 35 shares, and one is paying 25 shares.
